optimizely/helpers/condition.py

```diff
@@ -55,21 +55,23 @@ class ConditionMatchTypes:
     SEMVER_LE: Final = 'semver_le'
     SEMVER_LT: Final = 'semver_lt'
     SUBSTRING: Final = 'substring'
+    QUALIFIED: Final = 'qualified'
 
 
 class CustomAttributeConditionEvaluator:
     """ Class encapsulating methods to be used in audience leaf condition evaluation. """
 
-    CUSTOM_ATTRIBUTE_CONDITION_TYPE: Final = 'custom_attribute'
+    CONDITION_TYPES: Final = ('custom_attribute', 'third_party_dimension')
 
     def __init__(
         self,
         condition_data: list[str | list[str]],
-        attributes: Optional[optimizely_user_context.UserAttributes],
+        user_context: optimizely_user_context.OptimizelyUserContext,
```

Comment:
Most of them look like non-public type/methods, but if you do not feel comfortable about possible breaking changes, let's leave them as is.  We can try to add new ones with consistent names (either user or user_context).  Not 100% but a general rule is that any type starting with "Optimizely" are public.

Reply:
If they are considered non-public, I think it's fine, as long as we call it out in the release notes. Especially since it's actually only `get_variation_for_rollout` that is currently using `user`.